lib/client.ts

```diff
@@ -50,6 +49,17 @@ class Client {
         Client.connections.set(key, this);
     }
 
+    private async __pingPong() {
+        const reqId = await this.ping();
+        return this
+            .read(reqId)
+            .catch(() => null)
+            .finally(() => {
+                if (this.con?.readyState === this.con.OPEN)
+                    this.__pingPong()
+            });
```

Comment:
i don't see where you keep sending ping every 20s and where you reconnect if no pong is received after 40s